main.tf

```diff
@@ -0,0 +1,124 @@
+resource "aws_lambda_function" "main" {
+  function_name = var.name
+  role          = aws_iam_role.lambda.arn
+
+  tags = var.lambda_tags
+
+  // All parameters below are passed through directly
```

Comment:
/me creys in no spread/**kwargs-like construction for Terraform

Reply:
Very much agreed